### PR TITLE
Phrasing, precisions and typos in CLI help

### PR DESCRIPTION
**Pull request by Tbaut**

### CODE REVIEW

**Comment by 5chdn** [comment]
Thanks for this, it will be a mess in combination with #9036 though :D

**Comment by Tbaut** [comment]
we can wait and amend/merge it later

**Comment by Tbaut** [comment]
@5chdn  looks like there is no conflict :D

File: parity/cli/mod.rs [hunk 1]
```diff
@@ -49,16 +49,16 @@ usage! {
...
 
 			CMD cmd_account_list {
-				"List existing accounts",
+				"List existing accounts of the given --chain (default: foundation)",
 			}
 
 			CMD cmd_account_import
 			{
-				"Import account",
+				"Import accounts from JSON UTC keystore files to the specified --chain (default mainnet)",
```
Comment (5chdn) [inline parity/cli/mod.rs]: mainnet? foundation?

File: parity/cli/mod.rs [hunk 2]
```diff
@@ -157,11 +157,11 @@ usage! {
 			"Manage signer",
 
 			CMD cmd_signer_new_token {
-				"Generate new token",
+				"Generate new token for the given --chain (default: foundation)",
```
Comment (5chdn) [inline parity/cli/mod.rs]: Generate a new **signer-authentication** token...

File: parity/cli/mod.rs [hunk 3]
```diff
@@ -157,11 +157,11 @@ usage! {
...
 			}
 
 			CMD cmd_signer_list {
-				"List",
+				"List the tokens from given --chain (default: foundation)",
```
Comment (5chdn) [inline parity/cli/mod.rs]: signer-authentication tokens?

File: parity/cli/mod.rs (hunk 1 above)
Comment (Tbaut) [reply parity/cli/mod.rs]: mainnet everywhere now.

File: parity/cli/mod.rs [hunk 4]
```diff
@@ -82,7 +82,7 @@ usage! {
 
 		CMD cmd_import
 		{
-			"Import blockchain",
+			"Import blockchain from a file to the given --chain database (default: foundation)",
```
[the remark this answers is not in the document]
Comment (Tbaut) [reply parity/cli/mod.rs]: it's indeed the data. Added it.

File: parity/cli/mod.rs [hunk 5]
```diff
@@ -99,7 +99,7 @@ usage! {
 
 			CMD cmd_export_blocks
 			{
-				"Export blocks",
+				"Export blocks into the given --chain database (default: foundation). This command requires the chain to be synced with --fat-db on.",
```
[the remark this answers is not in the document]
Comment (Tbaut) [reply parity/cli/mod.rs]: - yup for the "from"
- I think 
>This command requires the chain to be synced with --fat-db on.

is correct

File: parity/cli/mod.rs [hunk 6]
```diff
@@ -198,7 +198,7 @@ usage! {
 
 		CMD cmd_restore
 		{
-			"Restore database from snapshot",
+			"Restore database of the given --chain (default: foundation) from snapshot",
```
[the remark this answers is not in the document]
Comment (Tbaut) [reply parity/cli/mod.rs]: fixed

File: parity/cli/mod.rs [hunk 7]
```diff
@@ -211,7 +211,7 @@ usage! {
 
 			CMD cmd_tools_hash
 			{
-				"Hash a file",
+				"Hash a file using Keccak-256 algorithm",
```
[the remark this answers is not in the document]
Comment (Tbaut) [reply parity/cli/mod.rs]: fixed

File: parity/cli/mod.rs (hunk 2 above)
Comment (Tbaut) [reply parity/cli/mod.rs]: fixed

File: parity/cli/mod.rs (hunk 3 above)
Comment (Tbaut) [reply parity/cli/mod.rs]: fixed

File: parity/cli/mod.rs (hunk 5 above)
[the remark this answers is not in the document]
Comment (dvdplm) [reply parity/cli/mod.rs]: works for me

**Pull request merged**